helm/alfresco-content-services/values.yaml

```diff
@@ -122,7 +122,9 @@ repository:
 apiexplorer:
   ingress:
     path: /api-explorer
-ooiService:
+alfresco-connector-ms365:
+  enabled: false
+# -- Enable/Disable Alfresco Collaboration Connector for Microsoft 365
   replicaCount: 2
   nodeSelector: {}
```

Comment:
Only specify the configuration that differs from the component chart AND the image & tag (so the updatecli pipelines are always deterministics). Remove everything else